nixos/modules/i18n/input-method/kime.nix

```diff
@@ -0,0 +1,49 @@
+{ config, pkgs, lib, generators, ... }:
+with lib;
+let
+  cfg = config.i18n.inputMethod.kime;
+  yamlFormat = pkgs.formats.yaml { };
+in
+{
+  options = {
+    i18n.inputMethod.kime = {
+      config = mkOption {
+        type = yamlFormat.type;
+        default = { };
+        example = literalExample ''
```

Comment:
I haven't used foreign input methods in a **long** time so my memory is rusty... What is the minimum configuration required here to make `kime` work? Would leaving this empty be enough? The documentation suggests it is... but I just wanted to confirm.

Reply:
Yes I works even if there is no config file

Reply:
Perfect. Thank you for confirming :+1: